### ISSUE TITLE
Error at Flow dataset import 

### ISSUE THREAD

**Issue by nadiagorchakova** [akvo/akvo-lumen#1374]
# Context

Molo reported an error while trying to load a dataset from the following Flow instance:

http://wai.akvoflow.org/ the survey WAI Bangladesh WASH SDG->Bangladesh Baseline 2018->Household Survey Final

# Problem or idea

The error he got:

![screen-shot-20180410-at-11](https://user-images.githubusercontent.com/21127166/38563452-34102c36-3cdd-11e8-9831-f0d52ce3a985.png)

I tried to reproduce it but got a different error (same Flow form)

<img width="435" alt="screen shot 2018-04-10 at 16 34 04" src="https://user-images.githubusercontent.com/21127166/38563481-472c43b8-3cdd-11e8-99e5-1fffa55bc49b.png">

# Solution or next step
@kardan could you please have a look?

**Comment by tangrammer** [comment]
I could recreate the problem with example [data](https://gist.github.com/tangrammer/a9f572e707294624290b7290636d2e59) 

https://github.com/akvo/akvo-lumen/blob/ef4d0b33dd3f845b406c6a50ed7a3ff437273ecf/backend/src/akvo/lumen/import/flow_common.clj#L96

```
IllegalArgumentException Don't know how to create ISeq from: java.lang.Double
	clojure.lang.RT.seqFrom (RT.java:550)
	clojure.lang.RT.seq (RT.java:530)
	clojure.lang.RT.first (RT.java:686)
	clojure.core/first--5106 (core.clj:55)
	clojure.core/first--5106 (core.clj:55)
	clojure.core/map/fn--5587 (core.clj:2747)
	clojure.lang.LazySeq.sval (LazySeq.java:40)
	clojure.lang.LazySeq.seq (LazySeq.java:49)
	clojure.lang.Cons.next (Cons.java:39)
	clojure.lang.RT.boundedLength (RT.java:1785)
	clojure.core/apply (core.clj:657)
	clojure.core/apply (core.clj:652)
nil
```

Summarising it's a data problem that doesn't fit to previous function expectations

**Comment by tangrammer** [comment]
```
java.lang.IllegalArgumentException: Don't know how to create ISeq from: java.lang.Integer
	at clojure.lang.RT.seqFrom(RT.java:550)
	at clojure.lang.RT.seq(RT.java:530)
	at clojure.core$seq__5124.invokeStatic(core.clj:137)
	at clojure.core$map$fn__5587.invoke(core.clj:2738)
	at clojure.lang.LazySeq.sval(LazySeq.java:40)
	at clojure.lang.LazySeq.seq(LazySeq.java:49)
	at clojure.lang.LazySeq.first(LazySeq.java:71)
	at clojure.lang.RT.first(RT.java:685)
	at clojure.core$first__5106.invokeStatic(core.clj:55)
	at clojure.string$join.invokeStatic(string.clj:180)
	at clojure.string$join.invoke(string.clj:180)
	at akvo.lumen.import.flow_v2$eval36343$fn__36344.invoke(flow_v2.clj:76)
	at clojure.lang.MultiFn.invoke(MultiFn.java:233)
	at akvo.lumen.import.flow_v3$render_response.invokeStatic(flow_v3.clj:43)
	at akvo.lumen.import.flow_v3$render_response.invoke(flow_v3.clj:34)
	at akvo.lumen.import.flow_v3$response_data$fn__36397.invoke(flow_v3.clj:53)
	at clojure.lang.ArrayChunk.reduce(ArrayChunk.java:58)
	at clojure.core.protocols$fn__7847.invokeStatic(protocols.clj:136)
	at clojure.core.protocols$fn__7847.invoke(protocols.clj:124)
	at clojure.core.protocols$fn__7807$G__7802__7816.invoke(protocols.clj:19)
	at clojure.core.protocols$seq_reduce.invokeStatic(protocols.clj:31)
	at clojure.core.protocols$fn__7835.invokeStatic(protocols.clj:75)
	at clojure.core.protocols$fn__7835.invoke(protocols.clj:75)
	at clojure.core.protocols$fn__7781$G__7776__7794.invoke(protocols.clj:13)
	at clojure.core$reduce.invokeStatic(core.clj:6748)
	at clojure.core$reduce.invoke(core.clj:6730)
	at akvo.lumen.import.flow_v3$response_data.invokeStatic(flow_v3.clj:48)
	at akvo.lumen.import.flow_v3$response_data.invoke(flow_v3.clj:45)
	at akvo.lumen.import.flow_v3$form_data$fn__36404.invoke(flow_v3.clj:69)
	at clojure.core$map$fn__5587.invoke(core.clj:2745)
	at clojure.lang.LazySeq.sval(LazySeq.java:40)
	at clojure.lang.LazySeq.seq(LazySeq.java:49)
	at clojure.lang.RT.seq(RT.java:528)
	at clojure.core$seq__5124.invokeStatic(core.clj:137)
	at clojure.core$map$fn__5587.invoke(core.clj:2738)
	at clojure.lang.LazySeq.sval(LazySeq.java:40)
	at clojure.lang.LazySeq.seq(LazySeq.java:49)
	at clojure.lang.RT.seq(RT.java:528)
	at clojure.core$seq__5124.invokeStatic(core.clj:137)
	at clojure.core$seq__5124.invoke(core.clj:137)
	at akvo.lumen.import$do_import.invokeStatic(import.clj:83)
	at akvo.lumen.import$do_import.invoke(import.clj:72)
	at akvo.lumen.import$handle_import_request$fn__36646.invoke(import.clj:101)
	at clojure.core$binding_conveyor_fn$fn__5476.invoke(core.clj:2022)
	at clojure.lang.AFn.call(AFn.java:18)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by tangrammer** [comment]
Closing this issue as will be tracked from https://github.com/akvo/akvo-flow/issues/2667

**Comment by janagombitova** [comment]
@nadiagorchakova I took a look at the issue as Juan pointed it out in the Flow repo. I took a look at the data and there is one question set as a cascade and 7 submissions, all from 17-01-2018 (the only submissions from that day) have answers to the question that do not fit the cascade structure.

This could be that the survey was not finalised at that point and the question was not set as a cascade so the user could put in whatever he/she wanted.

https://user-images.githubusercontent.com/12456965/40615879-fb85f00e-6288-11e8-9ea4-35114a3e39b2.png

I could not find the link to reamaze, but sent Molo a message directly.
